ci/CLOUD_BUILD.md

```diff
@@ -0,0 +1,57 @@
+# Using Cloud Build to install Spinnaker for GCP
+
+## A note about Shared VPC support
+
+You can't use Cloud Build to install Spinnaker for GCP with a shared VPC. For Shared VPC support, conduct the [setup in Cloud Shell](https://cloud.google.com/docs/ci-cd/spinnaker/spinnaker-for-gcp).
+
+## Service account
+
+To install Spinnaker for GCP, you need to grant the [Cloud Build service account](https://console.cloud.google.com/cloud-build/settings) the following roles:
+
+- Cloud Functions Developer - roles/cloudfunctions.developer
+- Compute Network Viewer - roles/compute.networkViewer
+- Kubernetes Engine Admin - roles/container.admin 
+- Create Service Accounts - roles/iam.serviceAccountCreator
+- Pub/Sub Editor - roles/pubsub.editor
+- Cloud Memorystore Redis Admin - roles/redis.admin
+- Service Usage Admin - roles/serviceusage.serviceUsageAdmin
+- Source Repository Administrator - roles/source.admin
+- Storage Admin - roles/storage.admin
+- Project IAM Admin - roles/resourcemanager.projectIamAdmin
+- Service Account User - roles/iam.serviceAccountUser
+
+You can enable these roles using the IAM UI or with [gcloud](https://cloud.google.com/sdk/gcloud/reference/projects/add-iam-policy-binding).
+
+## Enable Cloud Resource Manager API
+
+For Cloud Build to successfully retrieve IAM policies, you must enable the Cloud Resource Manager API. Visit this URL, substituting your project number.
```

Comment:
Could give guidance on where to locate project number.

Reply:
Actually, is this supposed to be "project id"?

Reply:
Originally, I used the project number to enable/disable but both seem to work. I will update to use project id and make this consistent with the rest of the documentation.